test/cuda/test_cudaops.py

```diff
@@ -61,6 +61,25 @@ def _create_fastgelu_test_model(domain='ai.onnx.contrib'):
         model = make_onnx_model(graph)
         return model
 
+    @staticmethod
+    def _create_fastgelu_test_model_f16(domain='ai.onnx.contrib'):
+        nodes = [
+            helper.make_node(
+                'FastGelu', ['x', 'bias'], ['y'],
+                domain=domain)
+        ]
+
+        input0 = helper.make_tensor_value_info(
+            'x', onnx_proto.TensorProto.FLOAT16, [])
```

Comment:
dumb question, which fp16 was tested here, MFloat16 or BFloat16?

Reply:
It is MFloat16,
For BFloat16, we need to test it by native cases since the type is not exposed via python.